config/formats.js

```diff
@@ -467,6 +467,1363 @@ exports.Formats = [
 			this.add('raw', '<center><p><a title="Digimon Showdown Players Guide" href="https://1drv.ms/b/s!AvoD6RnUzzMvgmLcX1rqT8GTnEVK" target="_blank"><img src="https://yheeqg.bn1303.livefilestore.com/y4mcqvreFTM4wIjnVqqSI98LZXT-lFFCxPBYHyHGZtWJTIGuah-spBzqvXbLiWvyJgDnrjSRFziff59ZLLBEtm_t3ZHHeKv9AY6Ml-gQpdGMhNmzaeSt0TD8wmivpfEW81jh93LSBvdXb7-cZUH2YNJwzwiyOTsy4L2dViyJqkiOCFDZhJRIPwPEWgmD7MDJnfafmoq6SN6gEfsAXMiHh529w?width=124&amp;height=42&amp;cropmode=none" alt="" width="124" height="42" /></a></p></center>');
 		},
 	},
+	{
+		name: "[Gen 7] Type Illusion",
+		section: "SpacialGaze Metagames",
+		mod: "gen7",
+		ruleset: ['[Gen 7] OU'],
+		banlist: ["Shedinja"],
+		allowTies: true,
```

Comment:
`allowTies`? Wont cause any issues if its there, but yeah probably best to remove it as were still working on ties on main. We might change how its marked.